subsys/bluetooth/controller/hci_driver.c

```diff
@@ -137,6 +137,12 @@ BUILD_ASSERT(!IS_ENABLED(CONFIG_BT_PERIPHERAL) ||
 	#define SDC_SCAN_SIZE 0
 #endif
 
+#if defined(SDC_MEM_INIT)
```

Comment:
I would be happy if we could use "INITIATOR" instead of "INIT" to avoid confusion